db/migrate/20200207104825_create_books.rb

```diff
@@ -0,0 +1,15 @@
+class CreateBooks < ActiveRecord::Migration[6.0]
+  def change
+    create_table :books do |t|
+      t.string :title, null: false
+      t.string :author, null: false
+      t.text :description
+      t.string :isbn10, limit: 10
+      t.string :isbn13, limit: 13
+      t.date :release_date
+      t.string :status, default: 'available'
+      t.references :user, foreign_key: true
```

Comment:
I think it'd be good to add a 'null: false' constraint here as a book will always depend on a user.

Reply:
You are absolutely right, thanks for this notice. I will add it.